### PR TITLE
[VS]: Fix test_swss_neighbor_syncup() problem seen with Linux 4.15 ke…

### PR DESCRIPTION
**Pull request by jipanyang**
…rnel env.

Signed-off-by: Jipan Yang <jipan.yang@alibaba-inc.com>

**What I did**
Fix test_swss_neighbor_syncup() failure
**Why I did it**
Seen problem with cache-state:STALE log check, and ip neigh change failure.
**How I verified it**
VS test
**Details if related**

### CODE REVIEW

**Comment by lguohan** [comment]
retest this please

File: tests/test_warm_reboot.py [hunk 1]
```diff
@@ -327,12 +327,12 @@ def check_syslog_for_neighbor_entry(dvs, new_cnt, delete_cnt, iptype):
     if iptype == "ipv4":
         (exitcode, num) = dvs.runcmd(['sh', '-c', 'grep neighsyncd /var/log/syslog| grep cache-state:NEW | grep IPv4 | wc -l'])
         assert num.strip() == str(new_cnt)
-        (exitcode, num) = dvs.runcmd(['sh', '-c', 'grep neighsyncd /var/log/syslog| grep cache-state:DELETE | grep IPv4 | wc -l'])
+        (exitcode, num) = dvs.runcmd(['sh', '-c', 'grep neighsyncd /var/log/syslog| grep \'cache-state:DELETE\|cache-state:STALE\' | grep IPv4 | wc -l'])
```
Comment (zhenggen-xu) [inline tests/test_warm_reboot.py]: This change looks good.

File: tests/test_warm_reboot.py [hunk 2]
```diff
@@ -545,10 +545,10 @@ def test_swss_neighbor_syncup(dvs):
 
     # add even nummber of ipv4/ipv6 neighbor entries to each interface
     for i in range(0, len(ips), 2):
-        dvs.runcmd("ip neigh change {} dev {} lladdr {}".format(ips[i], intfs[i%2], macs[i]))
+        dvs.runcmd("ip neigh add {} dev {} lladdr {}".format(ips[i], intfs[i%2], macs[i]))
```
Comment (zhenggen-xu) [inline tests/test_warm_reboot.py]: Do you know starting from which kernel we saw this change?

**Comment by jipanyang** [comment]
Will revisit the issue after code refactoring of neighsyncd warm restart reconciliation.

**Comment by lguohan** [comment]
covered in #649

**Pull request closed without merging**